src/package/vnode.ts

```diff
@@ -33,6 +33,7 @@ export interface VNodeData {
   ns?: string // for SVGs
   fn?: () => VNode // for thunks
   args?: any[] // for thunks
+  is?: string // for custom elements v1
```

Comment:
What does the `v1` here mean? Perhaps it should be removed?

Reply:
I assume it's referring to the spec (custom elements v0 is deprecated) https://caniuse.com/?search=custom%20elements

Reply:
Yeah, it's for spec.